internal/lib/rename.go

```diff
@@ -15,7 +15,7 @@ import (
 const configFile = "config.json"
 
 // ContainerRename renames the given container
-func (c *ContainerServer) ContainerRename(container, name string) error {
+func (c *ContainerServer) ContainerRename(container, name string) (errRet error) {
```

Comment:
we flipped the name. It should be `retErr`

Reply:
@mrunalp 
Comment addressed.